### ISSUE TITLE
using yolo for other environments

### ISSUE THREAD

**Issue by adiv5** [mbaske/yolo-unity#1]
I find the work as great.
I wanted to know how shall someone use the yolo model for their specific unity environment and scenes.
What are the steps to apply yolo for custom environments rather than default provided one.

Thanks

**Comment by mbaske** [comment]
Hi @adiv5 - It depends on what kind of objects you want to detect. Do any of the [pre-trained Yolo models](https://github.com/AlexeyAB/darknet/#pre-trained-models) include those objects? If yes, then you would just build the project using the model that best fits your needs (I included v3-tiny by default). 
If not, then you would need to create your own annotated training data. I haven't tried training Yolo myself yet, but I suppose you'd have to render and export a number of images from your Unity environment and create the corresponding labels. Please see https://github.com/AlexeyAB/darknet/#how-to-train-to-detect-your-custom-objects

**Comment by rash1994** [comment]
Hie @mbaske - Did you check the performance in android phone ? 
how was it ?

**Comment by mbaske** [comment]
> Hie @mbaske - Did you check the performance in android phone ?
> how was it ?

I didn't, it's just a windows experiment for now.
